tech/tech/domain/entities/orders.py

```
@@ -0,0 +1,19 @@
+from enum import Enum
+from typing import List, Optional
+
+class OrderStatus(str, Enum):
+    RECEIVED = 'RECEIVED'
+    PREPARING = 'PREPARING'
+    READY = 'READY'
+    FINISHED = 'FINISHED'
+
+
+class Order:
+    def __init__(self, total_price: float, product_ids: List[int], status: OrderStatus, id: Optional[int] = None):
```

Ruff (0.8.2)

Line too long (114 > 79)

(E501)